msteams-platform/concepts/deploy-and-publish/apps-access.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
I do not think we need this page. This is more about "using" the app, which comes after the dev publishes the app and is covered in our end user docs: https://support.microsoft.com/en-us/office/apps-and-services-cc1fba57-9900-4634-8306-2360a40c665b?ui=en-us&rs=en-us&ad=us.